pyccel/codegen/printing/ccode.py

```diff
@@ -2772,15 +2785,19 @@ def _print_ListPop(self, expr):
         else:
             return f'{c_type}_pull({list_obj})'
 
+    def _print_ListReverse(self, expr):
+        class_type = expr.list_obj.class_type
+        c_type = self.get_c_type(class_type)
+        list_obj = self._print(ObjectAddress(expr.list_obj))
+        return f'{c_type}_reverse({list_obj});\n'
```

Comment:
FYI: https://github.com/stclib/STC/commit/ef322aeb21c682a02253ce6df524307d142294c7
for after #2162 (which is blocked by #1969)